manifests/2.13.0/opensearch-2.13.0-concurrent-search-test.yml

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
I think we can remove this manifest until it is required for next version. Jenkins is getting overloaded these days due to release activities so just a caution